Packs/EclecticIQ/Integrations/EclecticIQIntelligenceCenterv3/EclecticIQIntelligenceCenterv3.py

```diff
@@ -2434,8 +2436,10 @@ def fetch_indicators(eiq_api):
             demisto.info("Feed id={} was fully ingested/updated.".format(str(item["id"])))
 
             return indicators_to_add
+        return None
```

Comment:
Rather not return None.
```suggestion
        return []
```

Reply:
done